website/docs/index.html.markdown

```diff
@@ -20,33 +20,35 @@ from a cloud provider.
 terraform {
   required_providers {
     virtualbox = {
-      source = "terra-farm/virtualbox"
-      version = "0.2.1"
+      source = "shekeriev/virtualbox"
+      version = "0.0.4"
     }
   }
 }
 
-# There are currently no configuration options for the provider itself.
+provider "virtualbox" {
```

Comment:
I would add that these configuration options are optional, so it doesn’t imply they are required.